src/java/org/apache/cassandra/service/snapshot/SnapshotManagerMBean.java

```diff
@@ -55,7 +55,17 @@ public interface SnapshotManagerMBean
     void clearSnapshot(String tag, Map<String, Object> options, String... keyspaceNames) throws IOException;
 
     /**
-     * Get the details of all the snapshots
+     * Get the details of all the snapshots. Options might be:
+     *
+     * <pre>
+     * no_ttl: "true" or "false"
+     * include_ephemeral: "true" or "false"
+     * keyspace: name of keyspace to get snapshots of
```

Comment:
From the server side logic, I see that keyspace, table and snapshot can be null. Should we document here what's the behavior on those cases?